collectors/o365/package.json

```diff
@@ -9,9 +9,9 @@
     "test": "JUNIT_REPORT_PATH=./test/report.xml nyc --reporter=text mocha --colors"
   },
   "devDependencies": {
-    "aws-sdk": "*",
-    "aws-sdk-mock": "*",
-    "jshint": "^2.9.5",
+    "aws-sdk": "^2.624.0",
```

Comment:
would you like to bump package version for this change?

Reply:
since those are dev dependencies, I don't think it's necessary. just trying very hard to banish the stars when I can.